helper-npm-install-peer-deps

```diff
@@ -11,10 +11,12 @@
 #
 # Why we pass the flags we do to `npm install`:
 #
-# --no-package-lock - Don't write a package-lock.json as
-#   we don't use it and it can cause issues with `npm ls`,
-#   which is run by subsequent tasks in CI:
-#   https://github.com/npm/npm/issues/19393
+# --no-package-lock - While we now use package-lock.json 
+#   in our repos, skip package-lock.json in this instance 
+#   as we're bypassing the normal npm install process     
+#   and don't want to accidentally modify the file in CI.
+#   Normal dependencies are still installed via `make install` 
+#   which does use the package-lock.
 #
 # --no-save - Don't update package.json - as we don't use
 #   package-lock.json, we use the checksum of package.json
```

Comment:
> as we don't use package-lock.json

This comment should also be updated to reflect that we do now use `package-lock.json` but still explain why in this context we need to use `--no-save`.